inputs/select.vue

```diff
@@ -59,7 +61,8 @@
   <ui-select
     v-if="hasOptions"
     :name="name"
-    :value="safeData"
+    :value="value"
```

Comment:
`safeData` wasn't doing anything for me so I changed this to match the Keen UI prop, lemme know if you think that's too dramatic a change